spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -226,6 +225,8 @@ public void onMessage(ConsumerRecord<Integer, String> message) {
 		container1.stop();
 		container2.stop();
 
+		props.put("auto.offset.reset", "earliest");
```

Comment:
We can use particular `ConsumerConfig` constants for such a case, e.g. this one is: `AUTO_OFFSET_RESET_CONFIG`.
But this is just a test, so doesn't matter. 
Just IMHO.

Merging...